int32 objNum
objInfo[] data
coeff[] coeff
lane[] lane
tunneloutpoint[] tunneloutpoint